dnguyen09/solvd-laba-test-autiomation
# Added 5 test cases for web testing

- **Test case 1: Test News Politic Categories**

Description: Verify that users can select a news category and view news related to that category.
Step:
	1. Open yahoo homepage by using chrome browser - The Yahoo homepage should load successfully.
	2. On the Yahoo homepage, locate and click on the "News" link on the top navigation menu - The user should be 
	   redirected to the Yahoo news page
	3. On the Yahoo News page, click on politics news category on the top navigation menu - The user should be redirected to 
	   the politics news page
	4. Verify if relevant news articles related to politics should be displayed
	
Test Result: the user should see the politics  news category page with relevant politics articles displayed on the page.

- **Test case 2: Test Weather City Search**

Description: Verify that users can search for a specific city's weather forecast and get according temperatures of the city in range.
Step:
	1. Open yahoo homepage by using chrome browser - The Yahoo homepage should load successfully.
	2. On the homepage, navigate to the "Weather" board or section
	3. Click on search location icon and put in city - the temperature according to the city should be loaded and not empty
	4. Verify if the temperature of the city is in range
	
Test Result: the weather temperatures for the specified city are displayed and in range.	

- **Test case 3: Test Sport Player Info**

Description: Verify that users can access and view accurate biographical information for a specific sport player.
Step:
	1. Open yahoo homepage by using chrome browser - The Yahoo homepage should load successfully.
	2. On the Yahoo homepage, locate and click on the "Sports" link in the top navigation menu - the user should be redirected to the Yahoo Sports page
	3. On the sport page, navigate to "NBA" at top navigation and click on "player" to open player page
	4. On the player page, navigate to search player name box and put in player name
	5. Click on the player link from the drop down 
	6. Verify if the Height/Weight and Born of the player are displayed accurately
	
Test Result: the sport player information is displayed accurately.

- **Test case 4: Test Invalid Login Email**

Description: Verify if the system handles invalid login email
Step:
	1. Open yahoo homepage by using chrome browser - The Yahoo homepage should load successfully.
	2. On the Yahoo homeage, click on the "Sign in" button at the top 
	3. On the login yahoo page, type invalid email to the "username" section and click "next" button
	4. Verify if the error message appears and if it provide correct message.

Test Result: the error message appears with correct message.

- **Test** case 5: Test Dynamical Loaded News**

Description: Verify that new news articles are dynamically loaded as the user scrolls down the page.
Step:
	1. Open yahoo homepage by using chrome browser - The Yahoo homepage should load successfully.
	2. On the Yahoo homepage, locate and click on the "News" link on the top navigation menu 
	3. On the Yahoo news page, scroll down the page to trigger dynamic news loading
	4. Verify that news articles are loaded
	
Test Result: new news articles are successfully loaded as the user scrolls down the page.
